### PR TITLE
Update TESTING.md

### PR DESCRIPTION
**Pull request by danielmarquespt**
Adds testing scenario for Symf/Local Keyword Search

### CODE REVIEW

**Review by mkondratek — APPROVED** [review]

File: TESTING.md [hunk 1]
```diff
@@ -320,6 +320,28 @@ To open the context menu:
 4. Trigger autocomplete in the editor.
 5. Verify: A similar notification is shown to the user.
 
+## Context
```
Comment (exigow) [inline TESTING.md:325]: I think we should add link with checkbox to this header, just like the rest.

File: TESTING.md [hunk 2]
```diff
@@ -320,6 +320,28 @@ To open the context menu:
...
+3. **Verify**: Local context is enabled:
+   
+  ![Screenshot 2024-02-06 at 15 18 18](https://github.com/sourcegraph/jetbrains/assets/7814431/11a68b1a-53a4-474e-97c7-74c18374beda)
+  
+5. Type: "what is squirrel?"
+6. **Verify**: You get an answer similar to:
+   > "Squirrel is a code intelligence service developed by Sourcegraph that uses tree-sitter for syntactic analysis of code. Some key things about Squirrel"
+8. Disable Local context
+   
+   ![Screenshot 2024-02-06 at 15 19 07](https://github.com/sourcegraph/jetbrains/assets/7814431/3c755039-e19e-4e58-a9d7-72ac1a381e16)
```
Comment (exigow) [inline TESTING.md]: We can remove this "screenshot" alt text.

**Comment by taylorsperry** [comment]
@danielmarquespt some feedback here and failing tests, but once you resolve I think this is good to merge 👍 Thank you!

File: TESTING.md (hunk 1 above)
Comment (danielmarquespt) [reply TESTING.md:325]: Done

File: TESTING.md (hunk 2 above)
Comment (danielmarquespt) [reply TESTING.md]: Done

**Pull request merged**